editor/src/components/canvas/canvas-strategies/strategies/absolute-duplicate-strategy.spec.browser2.tsx

```diff
@@ -44,7 +52,7 @@ describe('Absolute Duplicate Strategy', () => {
       'await-first-dom-report',
     )
 
-    FOR_TESTS_setNextGeneratedUid('hello')
+    FOR_TESTS_setNextGeneratedUids(['hello', 'hello', 'hello', 'hello', 'hello', 'hello'])
```

Comment:
hello! :D is it legal to generate the same UID? :D other tests usually do something like "dummy1", "dummy2" ... "hello"

Reply:
great find! It turns out that I was a bit too heavy-handed on refactoring, and accidentally dropped the code that saved the newly generated UIDs in `customStrategyState`. Without this, every time `apply` was called, a fresh UID was created, which bubbled up in tests as a need to generate the same UID multiple times, so the UID in the expected DOM is the desired one. Looking on the bright side, `FOR_TESTS_setNextGeneratedUid` is also a way to assert that only one new UID is generated, and it worked very well in this case.

Fixed in https://github.com/concrete-utopia/utopia/pull/3359/commits/ebfff98ed56674009b6eb9416ee785554b007884